exercises/bottle-song/.deprecated

```diff
@@ -0,0 +1,3 @@
+**NOTE: This exercise has been deprecated**
```

Comment:
Has this been deprecated? I thought this was bottle-song, which it refers to using instead on the very next line.

Reply:
Well spotted. So shall I move this to the beer-song folder?

Reply:
Yes, I believe that would be the right location.

Reply:
I moved it from the bottle-song folder to the beer-song folder.